code/script.js

```diff
@@ -1,51 +1,246 @@
-// Variables that point to selected DOM elements
-const chat = document.getElementById('chat')
+// ----- Variables that point to selected DOM elements -----
+
+const chat = document.getElementById('chat');
+const ticketBtn = document.getElementById('ticket-btn');
+const chatbot = document.getElementById('chatbot');
+const inputWrapper = document.getElementById('input-wrapper')
+const formInputField = document.getElementById('name-form');
+const inputField = document.getElementById('name-input');
+const ticketBtnContainer = document.getElementById("ticket-btn-container");
+const sendBtn = document.getElementById('send-btn');
+
 
 // If you need any global variables that you can use across different functions, declare them here:
+let customerName = "";
+let movieOption = "";
+let showtimeOption = "";
+let ticketOption = "";
+let price = "";
+
+const reservation = {
+  type: ["Adult", "Student", "Child"],
+  movies: ["Barbie (2023)", "Oppenheimer", "Teenage Mutant Ninja Turtles: Mutant Mayhem", "The Little Mermaid (2023)", "Aja baja Alfons Åberg", "Meg 2: The Trench"],
+  showtimesAvailable: ["12:00 PM", "3:00 PM", "6:00 PM"],
+};
+
+const ticketPrices = {  // in us$
+  adultPrice: [11],
+  studentPrice: [8],
+  childPrice: [4],
+};
+
 
+/* ----- Declare your functions after this comment ----- */
 
-// Declare your functions after this comment
+// Clears the input field.
+const clearInputField = () => {
+  inputField.value = "";
+};
 
-// This function will add a chat bubble in the correct place based on who the sender is
+// Removes all child elements from a specified parent element.
+function removeChildrenFunction() {
+  while (inputWrapper.hasChildNodes()) {
+    inputWrapper.removeChild(inputWrapper.firstChild);
+  }
+};
+
+// Refreshes the page.
+const reloadPage = () => {
+  setTimeout(() => {
+    location.reload();
+  }, 3000);
+};
+
+// Chat sound.
+const playAudio = () => {
+  const notification = document.getElementById('notification');
+  notification.volume = 0.6; // Set the volume to 60%
+  notification.play();
+};
+
+// Will add a chat bubble in the correct place. ShowMessage function takes arguments "message" and "sender".
```

Comment:
You're adding nice comments which makes it easier for me to follow logic of the code.